include/seqan3/alignment/aligned_sequence/debug_stream_alignment.hpp

```diff
@@ -98,7 +98,7 @@ namespace seqan3
 template <typename char_t, typename alignment_t>
 //!\cond
     requires (detail::debug_streamable_tuple<alignment_t> &&
-              detail::all_satisfy_aligned_seq<detail::tuple_type_list_t<remove_cvref_t<alignment_t>>>)
+              detail::all_model_writable_aligned_seq<detail::tuple_type_list_t<remove_cvref_t<alignment_t>>>)
```

Comment:
this one would be enough to require `all_model_aligned_seq`

Reply:
Thanks for catching this. 
Also: its `writable` not `writeable` right ?

Reply:
Yeah, I think `writable` is more correct

* https://www.drupal.org/project/drupal/issues/2898947
* https://github.com/trailblazer/representable/issues/188
* https://www.merriam-webster.com/dictionary/writeable (does not know that)